unidist/core/backends/mpi/core/local_object_store.py

```diff
@@ -277,6 +277,9 @@ def cache_serialized_data(self, data_id, data):
         data : object
             Serialized data to cache.
         """
+        # Copying is necessary to avoid corruption of data obtained through out-of-band serialization,
+        # and buffers are marked read-only to prevent them from being modified.
```

Comment:
```suggestion
        # We make a copy to avoid data corruption obtained through out-of-band serialization,
        # and buffers are marked read-only to prevent them from being modified.
        # `to_bytes()` call handles both points.
```

Reply:
done